docker-entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/ash
+
+set -euo pipefail
+
+echo "==> $(date +%H:%M:%S) ==> Migrating Django models..."
+python src/manage.py migrate --noinput
+
+echo "==> $(date +%H:%M:%S) ==> Applying fixtures..."
+python src/manage.py loaddata initial_default_apps.json
```

Comment:
Will this be removed in next versions?

Reply:
Do you think it's better for us to load the fixture independently of the deployment? I can remove it on the context of this PR

Reply:
Removed here 46cc4dc

Reply:
Yes, I think so. If not we can find surprises if we forget about it